src/helm/benchmark/presentation/run_specs_chat_gpt.conf

```diff
@@ -96,35 +96,37 @@ entries: [
   # browse the internet or access any new information that may have become available since my training."
   # to many of the questions. However it still seems to do pretty well on certain subjects
   # like `author` and `plantiff`.
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=plaintiff", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=place_of_birth", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=medical_condition_treated", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=instance_of", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=part_of", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=currency", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=position_held", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=author", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=discoverer_or_inventor", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=symptoms_and_signs", priority: 2}
+  # Skip for now since it's more expensive to run (num_outputs=5)
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=plaintiff", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=place_of_birth", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=medical_condition_treated", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=instance_of", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=part_of", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=currency", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=position_held", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=author", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=discoverer_or_inventor", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=symptoms_and_signs", priority: 2}
 
 
   ##### Reasoning #####
 
   # Got response: "I'm sorry, but I cannot solve the problem as you have not provided
```

Comment:
can you post these in the reasoning channel and see if those folks have any thoughts/they might be interested to hear